mmdet/models/backbones/pvt.py

```diff
@@ -275,7 +276,7 @@ def __init__(self, pos_shape, pos_dim, drop_rate=0., init_cfg=None):
         self.drop = nn.Dropout(p=drop_rate)
 
     def init_weights(self):
-        trunc_normal_init(self.pos_embed, std=0.02)
+        trunc_normal_(self.pos_embed, std=0.02)
```

Comment:
What is the difference between the two?

Reply:
```trunc_normal_init``` requires input module must be ```nn.Module```, neither ```torch.Tensor``` nor ```nn.Parameter```. If the input is ```torch.Tensor```, the tensor will not be init as it should be.

Reply:
for more details, please refer to [```trunc_normal_init```](https://github.com/open-mmlab/mmcv/blob/c9d464c36094f062add8183d5e4bcef570f83cbc/mmcv/cnn/utils/weight_init.py#L73)

Reply:
> What is the difference between the two?

 `trunc_normal_init` can only init module with weight and bias, not nn.Parameter. The position embedding is not initialized correctly now.

Reply:
O, I get it. It is best to add a note to explain the precautions, because it is very easy to confuse.

Reply:
Maybe it's better to add some warning in ```MMCV```, two implementations of ```trunc init``` are really really confused.

Reply:
> Maybe it's better to add some warning in `MMCV`, two implementations of `trunc init` are really really confused.

Agree. Welcome to create a PR to MMCV.